src/index.js

```diff
@@ -14,15 +14,14 @@ let initModule = false;
  * @param  {string} [config.LOGS_TOKEN] Token used by internal logger to send logs to
  * @return {Object}        The configured providers
  */
-module.exports = function(config) {
-  //if no param is provided, we want to return the cached modules
+module.exports = (config) => {
+  // if no param is provided, we want to return the cached modules
   if (!config) {
```

Comment:
It may cause issue if you want to have two instances:
  - one with a transport (logentries)
  - one without transport

You wont be able to create the one without transport unless you create it first in code, this may be tricky to understand and should be well documented